core/src/main/java/org/apache/carbondata/core/readcommitter/LatestFilesReadCommittedScope.java

```diff
@@ -42,10 +39,27 @@
 public class LatestFilesReadCommittedScope implements ReadCommittedScope {
 
   private String carbonFilePath;
+  private String segmentId;
   private ReadCommittedIndexFileSnapShot readCommittedIndexFileSnapShot;
   private LoadMetadataDetails[] loadMetadataDetails;
 
-  public LatestFilesReadCommittedScope(String path)  {
+  /**
+   * a new constructor of this class, which supports obtain lucene index in search mode
```

Comment:
Better not to mention search mode. Describe what this constructor will do.

Reply:
ok, remove it